pom.xml

```diff
@@ -2012,7 +2012,7 @@
         <plugin>
           <groupId>net.alchim31.maven</groupId>
           <artifactId>scala-maven-plugin</artifactId>
-          <version>3.2.2</version>
+          <version>3.3.1</version>
```

Comment:
Darn, this seems to cause a failure when Maven and zinc are used, only -- can't find javac because it looks in a JRE directory. I'll revert this if I can't see an immediate solution. that's why it wasn't caught by me or the PR builder, I think.

Reply:
No easy solution; related to https://github.com/davidB/scala-maven-plugin/issues/161 I'm reverting this part of the change.